__tests__/commands/upgrade.js

```diff
@@ -31,6 +31,11 @@ const expectInstalledDevDependency = async (config, name, range, expectedVersion
   await _expectDependency('devDependencies', config, name, range, expectedVersion);
 };
 
+const expectInstalledTransientDependency = async (config, name, range, expectedVersion) => {
```

Comment:
i think transient is not the right word here. It should be transitive.

Reply:
whoopsies 😆 thanks for noticing that.